### PR TITLE
Don't emit urllib3 warnings to stderr

### PR DESCRIPTION
**Pull request by jgrund**
Disable insecure requests warning from urllib3 so we don't break our syslog handler.

This (disabled) warning is expected due to our use of self-signed certificates when we communicate between the agent and manager.

Also scope iml-device-scanner < 2.0.0.

Signed-off-by: Joe Grund <joe.grund@intel.com>

---
This change is [Reviewable](https://reviewable.io/reviews/intel-hpdd/intel-manager-for-lustre/482)

### CODE REVIEW

File: chroma-agent/chroma-agent.service [hunk 1]
```diff
@@ -9,5 +9,6 @@ WantedBy=multi-user.target
 Type=simple
 ExecStart=/usr/bin/chroma-agent-daemon
 Restart=on-failure
+SyslogFacility=daemon
```
Comment (brianjmurrell) [inline chroma-agent/chroma-agent.service]: Will this actually make any difference?  It already defaults to `daemon`.

Comment (jgrund) [reply chroma-agent/chroma-agent.service]: yeah... looks like `KeyError` it's still there. 

Wonder if I can get this silenced somehow:

```
Feb 27 12:18:02 localhost chroma-agent-daemon: /usr/lib/python2.7/site-packages/urllib3/connectionpool.py:769: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.org/en/latest/security.html
Feb 27 12:18:02 localhost chroma-agent-daemon: InsecureRequestWarning)
```

Comment (brianjmurrell) [reply chroma-agent/chroma-agent.service]: Yes PLEASE!

Comment (jgrund) [reply chroma-agent/chroma-agent.service]: I think I found a way, working on it now.

Comment (jgrund) [reply chroma-agent/chroma-agent.service]: Tested locally, looks like latest patch does the trick.

File: chroma-agent/chroma_agent/agent_daemon.py [hunk 2]
```diff
@@ -22,6 +22,10 @@
 from chroma_agent.lib.agent_startup_functions import agent_daemon_startup_functions
 from chroma_agent.lib.agent_teardown_functions import agent_daemon_teardown_functions
 
+# Disable insecure requests warning
+# So we don't break our syslog handler.
```
[the remark this answers is not in the document]
Comment (jgrund) [reply chroma-agent/chroma_agent/agent_daemon.py]: Sure. I do have a comment already, but I'll expand it a little more.

**Pull request merged**